app/controllers/spree/admin/recurrings_controller.rb

```diff
@@ -31,7 +31,12 @@ def update
       end
 
       def destroy
-        @recurring.restrictive_destroy
+        if @recurring.restrictive_destroy
+          flash[:success] = "Recurring has been deleted."
```

Comment:
@praveendhawan I think these are js requests hence we should use `flash.now` instead.